docs/Reporting.md

```diff
@@ -1,58 +1,66 @@
 # FCA: Reporting
 
-TODO: add intro
+FCA is pre-configured to generate detailed mochawesome JSON reports following the completion of [sanity tests](./Execution.md). This functionality not only provides users with immediate test results but also enables sending these reports to AWS or any other URL endpoint for further processing and analysis. Reporting needs will vary on a case-by-case basis and users have the flexibility to utilize the generated mochawesome report JSON in various ways, including transforming the JSON data into an HTML report.
```

Comment:
let's link directly to the `[sanity tests](./Execution.md#sanity)`

Reply:
Cleaned all of this up